Sources/Swift/Tools/UIRedactBuilder.swift

```diff
@@ -141,46 +133,67 @@ class UIRedactBuilder {
         return image.imageAsset?.value(forKey: "_containingBundle") == nil
     }
     
-    private func mapRedactRegion(fromView view: UIView, to: CALayer, redacting: inout [RedactRegion], area: CGRect, redactText: Bool, redactImage: Bool) {
-        let rectInWindow = (view.layer.presentation() ?? view.layer).convert(view.bounds, to: to)
-        guard (redactImage || redactText) && area.intersects(rectInWindow) && !view.isHidden && view.alpha != 0 else { return }
+    private func mapRedactRegion(fromView view: UIView, redacting: inout [RedactRegion], rootFrame: CGRect, redactOptions: SentryRedactOptions, transform: CGAffineTransform) {
+        guard (redactOptions.redactAllImages || redactOptions.redactAllText) && !view.isHidden && view.alpha != 0 else { return }
```

Comment:
hmm, so if we set both redactAllImages and redactAllText to `false`, but added a custom view or view type to redact, we'd bail from this method entirely and wouldn't redact anything? Or did I get this wrong?

Reply:
You're correct. This will be replaced in the upcoming [PR](https://github.com/getsentry/sentry-cocoa/pull/4327)